src/main/java/hw_11/Task2.java

```diff
@@ -0,0 +1,100 @@
+package hw_11;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Task2 {
+
+    private static Scanner scanner = new Scanner(System.in);
```

Comment:
can be final(don't forget about const naming)

Reply:
fixed